routes/matches/matches.js

```diff
@@ -460,14 +468,28 @@ router.get("/:match_id", async (req, res, next) => {
       req.user !== undefined &&
       (matchRow[0].user_id == req.user.id || Utils.superAdminCheck(req.user))
     ) {
-      sql = "SELECT * FROM `match` where id=?";
+//      sql = "SELECT * FROM `match` where id=?";
+	sql =  "SELECT `match`.id, `match`.user_id, server_id, team1_id, team2_id, winner, " +
+        "team1_score, team2_score, team1_series_score, team2_series_score, " +
+        "team1.name as team1_string, team2.name as team2_string, cancelled, forfeit, start_time, end_time, " +
+        "max_maps, title, skip_veto, private_match, enforce_teams, min_player_ready, " +
+        "season_id, is_pug, map_sides, gs.display_name FROM `match`" +
+        "LEFT JOIN team team1 ON team1.id = `match`.team1_id " +
+        "LEFT JOIN team team2 ON team2.id = `match`.team2_id " +
+        "LEFT JOIN game_server gs on gs.id = match.server_id " +
+        "where `match`.id = ?";
+
     } else {
       sql =
-        "SELECT id, user_id, server_id, team1_id, team2_id, winner, " +
+        "SELECT `match`.id, `match`.user_id, server_id, team1_id, team2_id, winner, " +
```

Comment:
If using joins, please try to follow convention as well. `SELECT ... FROM `match` as mtch`.

Reply:
Using "match" instead of `match` causing an SQL error, i will update with mtch alias as soon as possible.

Reply:
Well, yes because match is a keyword, so you can use `FROM \`match\` as mtch` will prevent that :)